src/xrCDB/xrCDB.cpp

```diff
@@ -43,7 +43,9 @@ MODEL::~MODEL()
 
 void MODEL::syncronize_impl() const
 {
-    Log("! WARNING: syncronized CDB::query");
+    #ifdef DEBUG
+        Msg("! WARNING: syncronized CDB::query");
+    #endif
```

Comment:
It's suboptimal to use `Msg` when formatting is not needed. In this case, the most optimal choice is `Log`.
Plus, formatting.
```suggestion
#ifdef DEBUG
    Log("! WARNING: syncronized CDB::query");
#endif
```